packages/flame/lib/src/sprite_batch.dart

```diff
@@ -254,15 +269,18 @@ class SpriteBatch {
     _batchItems.clear();
   }
 
+  // Used to not create new paint objects in [render] on every tick.
+  final Paint _emptyPaint = Paint();
+
   void render(
     Canvas canvas, {
     BlendMode? blendMode,
     Rect? cullRect,
     Paint? paint,
   }) {
-    paint ??= Paint();
+    paint ??= _emptyPaint;
```

Comment:
```suggestion
    paint ??= const Paint();
```
Then we don't need the field :)

Reply:
It still creates a new const object right? Or does dart have a heap with const objects internally?

Reply:
Dart will create only a single instance during the execution and use everywhere that has `const Paint()`

Reply:
Great, that'll look much better too :)

Reply:
Ah sadly `Paint` doesn't have a `const` constructor.

Reply:
Oh no :( nvm then 😅